### DISCUSSION TITLE
stability of computation related to permeability

### DISCUSSION THREAD

**Discussion by hekungeo** [Anura3D/Anura3D_OpenSource#55] [Q&A] [answered]
Dear all, 
Hi,
The attached figure shows the soil-water curve of soil. The Anura3D is conducted for the landslide runout.
The permeability decreases when the suction is very high. Is there any method that can update the permeability of the soil with the degree of saturation, or it is kept constant during Anura3D computation? and what is the criterion for the stability of the numerical scheme in the case of low-permeable soils?
![image](https://user-images.githubusercontent.com/105496180/209151448-79591f7d-6b5f-4ab1-9ba9-336995d62177.png)

**Comment by zhangfengeo** [comment]
Maybe you can choose "unsaturated material " in the GID,and then set VG model parameters of your SWCC,similarly,the constant parameter or Hillel model can also be chose as the hydraulic conductivity curve.

**Comment by wanlida** [comment]
感谢您的来件，本人已确定收到，48小时内会尽快回复您

**Comment by ayerrocolom** [comment, marked as answer]
Hello!
When using unsaturated formulation, you can also update the hydraulic conductivity depending on the degree of saturation. In the code, you can define three options: HCC_CONSTANT, HCC_HILLEL, or HCC_MUALEM. These are not defined in GiD yet but can be hard coded in the GOM file. The format of the input variables can be found in the Module "Read MaterialData.FOR" in the subroutine "ReadMaterial_v2021"

**Reply by CPLA81192** [reply]
hi, when I simulated rainfall infiltration, the calculation speed became very slow after changing the permeability coefficient. Whether the permeability increases or decreases, there is the same situation, may I ask what causes this
